melodic/distribution.yaml

```diff
@@ -6075,6 +6075,18 @@ repositories:
       type: git
       url: https://github.com/NiryoRobotics/niryo_one_ros_simulation.git
       version: master
+  nmc_nlp_lite-release:
+    doc:
+      type: git
+      url: https://github.com/nmcbins/nmc_nlp_lite-release.git
+      version: 0.0.5
```

Comment:
This should point to the release repository, and must point to a branch (not a tag).

Reply:
Thank you so much! I will fix it and retry!